### ISSUE TITLE
No alert level 4

### ISSUE THREAD

**Issue by ahurford** [canmod/nfld-macpan#3]
Hey @stevencarlislewalker I was trying to learn how to do a scenario, and I decided to try to see what would happen if the restrictions on Jan 4, 2022, in NL never happened.

The results are really interesting:
- Data/fit: NL had a two-peaked prolonged wave 5
- Scenario: without the restrictions, NL would have instead had a wave 5 more similar to other provinces (higher peak and over quicker).

I think I should have put this all in a folder - making a bit of a mess.

My coding is sloppy - do you have any tricks to make it better?

I like this simulation a lot. In NL, everyone is very confused because we hear about what is happening in ON, and think we are on the same timeline. It's really interesting to see that we would have been on a similar timeline had it not been for the Alert level 4 in January.

**Comment by stevencarlislewalker** [comment]
I'm still sorting through this ... sorry for the delays.  In general I think it looks great, and I'm working to clean it up and verify correctness.

**Comment by ahurford** [comment]
@stevencarlislewalker some data quality issues have come to my attention, so I am working on cleaning that up now.

**Comment by stevencarlislewalker** [comment]
Thanks @ahurford.  I was working a little as well, but I'll stop so we don't diverge so much.  I'm also going to post my thoughts on repo organization.

**Comment by stevencarlislewalker** [comment]
Hi @ahurford, I believe that we should have the following folder structure:
```
README.md
functions.R
get_data.R
vax_par_defaults.csv
[and other scripts and files not associated with a particular model/scenario]
initial_model
    initial_model.R -- script to fit/save the model and make/save the forecasts/scenarios
    initial_model.Rmd -- loads the saved objects produced by initial_model.R and produces a report
    initial_model.md -- report to look at on github
    initial_model_files -- files to support the report on github
        figure-html
            unnamed-chunk-7-1.png
            ...
        saved-objects -- where initial_model.R writes to
            fit.rds
            fitted_data.rds
            scenario.rds
            observed_data.rds
            ...
no_level_4
    no_level_4.R
    no_level_4.Rmd
    no_level_4.md
    no_level_4_files
        figure-html
            unnamed-chunk-7-1.png
            ...
        saved-objects
            fit.rds
            ...
another_model
    another_model.R
    another_model.Rmd
    another_model.md
    another_model_files
    ...
yet_another_model
...
```

Let me know when you are finished working on the data quality issue and I'll put everything in this format ... unless you disagree.  But the nice thing about the above is that it keeps the different modelling attempts separate so that we don't confuse ourselves about what modelling scripts led to each report.

**Comment by ahurford** [comment]
Hi @stevencarlislewalker I have finished working on the data quality issue. I was going to fit to a subset of reported cases, but then I ended up not fitting to cases at all. I think it works well in terms of where the breakpoints are and the fit that we get from just hospitalizations and deaths. It's fairly impossible to fit these data (even when just hospitalizations & deaths are fit) without those breakpoints, which emphasizes the importance of those restrictions in shaping NL's Omicron wave.

I have left variant.Rmd in a mess. I will work on that after you move it.

I also changed the plot appearance by editing in functions.R

**Comment by ahurford** [comment]
@stevencarlislewalker is it okay for me to release some of this on my websites? What is a workflow for doing that (just thinking that probably you aren't working on this over Easter)? [I know I'm not really there yet and have a bit of mess in the code in several places!]

**Comment by stevencarlislewalker** [comment]
It is really your call @ahurford.  If you want to be really careful you could mark it as a draft at the top of the page like this:

![](https://img.shields.io/static/v1.svg?label=Status&message=Draft&color=blue)

If you have sufficient control over your website, you can just copy the resulting html file over to your site.

I'm going to try to get to this before the end-of-day, but things are a little tight.

**Comment by ahurford** [comment]
Yes, that's good - I just wasn't sure to what extent I needed sign-off, or if it was okay for me to post this somewhere else. Thanks for all your help!
